source/frontend/src/components/maps/leaflet/Control/Search/PropertyQuickInfoContainer.tsx

```diff
@@ -0,0 +1,154 @@
+        setOwnerNames(names.join(', '));
+      }
+    },
+    [getLtsaExecute],
+  );
+
+  useEffect(() => {
+    getOwnerInfo(locationInfo?.PID);
+  }, [getOwnerInfo, locationInfo?.PID]);
+
+  const onViewPropertyInfo = useCallback(() => {
+    pathGenerator.showPropertyByPid(locationInfo.PID);
+  }, [locationInfo?.PID, pathGenerator]);
+
+  const onZoomToBounds = useCallback(() => {
+    if (exists(locationInfo?.SHAPE)) {
+      const bounds = geoJSON(locationInfo.SHAPE).getBounds();
+
+      if (exists(bounds) && bounds.isValid()) {
+        requestFlyToBounds(bounds);
+      }
+    } else if (exists(mapLocationFeatureDataset?.location)) {
+      requestFlyToLocation(mapLocationFeatureDataset.location);
+    }
+  }, [
+    locationInfo?.SHAPE,
+    mapLocationFeatureDataset?.location,
+    requestFlyToBounds,
+    requestFlyToLocation,
+  ]);
+
+  const showViewPropertyInfo = useMemo(
+    () => isValidString(locationInfo?.PID) && !hasMultipleProperties,
+    [hasMultipleProperties, locationInfo?.PID],
+  );
+
+  return (
+    <>
+      <StyledHeaderRow noGutters>
+        <Col xs="1">
+          {showViewPropertyInfo && (
+            <TooltipWrapper
+              tooltipId={`property-quick-info-view-property`}
+              tooltip={'View Property Information'}
+            >
+              <FaEye size={18} title="Zoom map" onClick={onViewPropertyInfo} />
+            </TooltipWrapper>
+          )}
+        </Col>
+        <Col xs="1">...</Col>
+        <Col xs="8" className="text-center">
+          Property
+        </Col>
+        <Col xs="1">
+          <TooltipWrapper tooltipId={`property-quick-info-zoom`} tooltip={'Zoom to property'}>
+            <FaSearchPlus size={18} title="Zoom map" onClick={onZoomToBounds} />
+          </TooltipWrapper>
+        </Col>
+        <Col xs="1">-</Col>
+      </StyledHeaderRow>
+      {!hasMultipleProperties && (
+        <StyledInfoWrapper>
+          <Row noGutters>
+            <Col>
+              <SectionField label={'PID'} labelWidth={{ xs: 12 }}>
+                {pidFormatter(locationInfo?.PID) ?? '-'}
+              </SectionField>
+            </Col>
+            <Col>
+              <SectionField label={'PIN'} labelWidth={{ xs: 12 }}>
+                {locationInfo?.PIN ?? '-'}
+              </SectionField>
+            </Col>
+            <Col>
+              <SectionField label={'Plan'} labelWidth={{ xs: 12 }}>
+                {locationInfo?.PLAN_NUMBER ?? '-'}
+              </SectionField>
+            </Col>
+          </Row>
+          <SectionField label={'Owners'} labelWidth={{ xs: 12 }}>
+            {ownerNames ?? '-'}
+          </SectionField>
+          <SectionField label={'Legal Description'} labelWidth={{ xs: 12 }}>
+            {locationInfo?.LEGAL_DESCRIPTION ?? '-'}
+          </SectionField>
+        </StyledInfoWrapper>
+      )}
+      {hasMultipleProperties && <div className="pt-8">Multiple properties found</div>}
```

Comment:
Hmm, how are we handling multiple properties for this component? does the map popup still exist for this use case maybe?

Reply:
Nvm, I see that logic was removed. Should I log a story to re-introduce that logic with the new quick-info workflow?

Reply:
we dont have a solution for the multi properties atm